pkg/controller/elasticsearch/remotecluster/remoteca/labels.go

```diff
@@ -0,0 +1,62 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package remoteca
+
+import (
+	"fmt"
+
+	esv1 "github.com/elastic/cloud-on-k8s/pkg/apis/elasticsearch/v1"
+	"github.com/elastic/cloud-on-k8s/pkg/controller/common"
+	"github.com/elastic/cloud-on-k8s/pkg/controller/elasticsearch/label"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	// RemoteClusterNamespaceLabelName used to represent the namespace of the RemoteCluster in a TrustRelationship.
+	RemoteClusterNamespaceLabelName = "remotecluster.k8s.elastic.co/namespace"
```

Comment:
this should probably have `elasticsearch.k8s.elastic.co` in the name.
Can we do `elasticsearch.k8s.elastic.co/remote-cluster: {"namespace": "ns", "name": "cluster-name"}`?

Reply:
I'm not sure we can put some json in a label (also I'm not sure it's easy to create a label selector against it)